benchbuild/project.py

```diff
@@ -27,12 +27,18 @@
 from plumbum import ProcessExecutionError, local
 from pygtrie import StringTrie
 
+import benchbuild as bb
+
 from benchbuild import signals
+from benchbuild.downloads.base import BaseSource
+from benchbuild.downloads.variants import VariantContext
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'benchbuild.downloads.variants'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)